submissions/TaniaBondarenko/friends-app/app.js

```diff
@@ -0,0 +1,130 @@
+const friends = document.querySelector(".friends");
+let friendsData;
+let arrForFilter;
+let isFiltered = false;
+let justMale;
+let justFemale;
+
+function makeContainerEmpty() {
+        let innerText = " "; 
+        friends.innerHTML = innerText.replace(innerText, " ");    
+}
+
+function fetchFriend() {
+        fetch(apiUrl)
+                .then(response => response.json())
+                .then((data) => friendsData = data.results)
+                .then(() => addFriends(friendsData))
+                .catch(err => {
+                        refreshPage();
+                });
+};
+
+function refreshPage() {
+        const wholePage = document.querySelector("body");
+        wholePage.setAttribute("class", "refresh");
+        wholePage.innerHTML = `<div>Ops...Something went wrong.</div>
+        <div>Refresh the page to start finding friends</div>
+        <button class="refresh_button">Click to refresh</button>` ;
+        document.querySelector(".refresh_button").addEventListener("click", fetchFriend);
+}
+
+ function addFriends(friendsToBeAdded) {
+      let fragment = document.createDocumentFragment();
+        friendsToBeAdded.forEach(friend=> {
+                let friendCard = document.createElement("div");
+               friendCard.setAttribute("class", "friend_card");
+                friendCard.innerHTML = `<div class="card_wrapper ${friend.gender}"><div class="name ">${`${friend.name.first} ${friend.name.last}`}</div>
+                                <div class="photo"><img src="${friend.picture.large}"></div>
+                                <div class="info_block">
+                                <div class="age">Age ${friend.dob.age}</div>
+                                <div class="place">${friend.location.city}</div>
+                                <div class="email"><a href="mailto:${friend.mail}" class="email_link">${friend.email}</a></div>
+                                <div class="gender">${friend.gender}</div>
+                                </div></div>
+                                `;
+                fragment.appendChild(friendCard);       
+        });
+              friends.appendChild(fragment);
+
+};
+
+document.querySelector(".sidebar").addEventListener("click", sortFriends);
+
+function sortFriends({ target }) {
+        chooseAppropriateFriends();
+        switch (target.className||target.id) {
+                case "ascendent_by_name":
+                        arrForFilter.sort((a, b) => a.name.first.localeCompare(b.name.first));
+                          redrawfriends(); 
+                        break;
+                case "descendent_by_name":
+                        arrForFilter.sort((a, b) => b.name.first.localeCompare(a.name.first));
+                          redrawfriends(); 
+                        break;
+                case "ascendent_by_age":
+                        arrForFilter.sort((a, b) => a.dob.age - (b.dob.age));
```

Comment:
Redundant brackets.

Please, be more attentive in such details. Or use prettier :D

Reply:
Prettier. Got it.